packages/flutter/lib/src/rendering/platform_view.dart

```diff
@@ -555,144 +553,6 @@ class _PlatformViewGestureRecognizer extends OneSequenceGestureRecognizer {
   }
 }
 
-typedef _GlobalToLocal = Offset Function(Offset point);
-
-// Composes a stream of PointerEvent objects into AndroidMotionEvent objects
-// and dispatches them to the associated embedded Android view.
-class _MotionEventsDispatcher {
```

Comment:
Moved to be a part of `AndroidViewController` and renamed as `_AndroidMotionEventConverter`.

Reply:
Sounds good